packages/react-app/src/lib/token.js

```diff
@@ -115,13 +116,23 @@ const fetchTokenDetailsFromContract = async token => {
 };
 
 export const fetchTokenDetails = async (bridgeDirection, token) => {
+  const mediatorAddress = getMediatorAddress(bridgeDirection, token);
+
+  if (NATIVE_CURRENCY_SYBMOLS.includes(token.symbol)) {
```

Comment:
Again no symbol check. Maybe set the mode in `constants` file itself and use that here. Or we could also skip this call to fetchTokenDetails for NATIVE tokens cause all info is already known.